README.md

```diff
@@ -1,5 +1,5 @@
 # Duffle: The CNAB Installer
-![Build Status](http://badges.technosophos.me/v1/github/build/deislabs/duffle/badge.svg?branch=master)
+![Build Status](http://badges.technosophos.me/v1/github/build/cnab/duffle/badge.svg?branch=master)
```

Comment:
The badge is still not working here - this should be a separate issue, not blocking this PR.

Reply:
Good call; I at least wanted to fix the typo I introduced (`cnab/duffle` -> `cnabio/duffle`) and in the process I updated to a new/running badge server.